src/pages/blog/baremetal-sig.md

```diff
@@ -0,0 +1,53 @@
+
+One of the great things of chatting in person is that it helps facilitate the use of the same words much faster. Body language allows us to identify and refine statements on the fly. This is harder to do even on video calls.
+
+It also allows us to see visual queues where more information is needed, and really allowed us to dive deep into various areas to help spread context and information.
+
+In any event, it was both a refreshing and powerful experience because we were able to obtain some data points!
+
+<br>
+For operators present:
+<br>
+* 33% are using the `redfish` hardware type in production.
+* 17% are using a Neutron ML2 plugin for automating port plugging in switches.
+* 15% are using Deploy Steps! - We had no idea!
+* 8% are running a long lived bifrost deployment. Yes, we're fairly sure they read the documentation where we stated this was not recommended usage.
+* 8% are operating over a 1,000 nodes in production with Ironic.
+* 4% are leveraging Cinder integration.
+* 4% reported use of the "ansible" deployment interface.
+* 3% are operating over 10,000 baremetal nodes managed by Ironic in production.
+* 2% indicated they are enforcing Secure Boot.
+* 2% indicated they are using Metal3 to drive Ironic.
+<br>
+One thing to keep in mind, when these gatherings occur and we're able to collect data from a random sampling of attendees at a conference, the data is skewed to the attendees who chose to be there.
+<br>
+A few observations:
+<br>
+* Increasing interest in ARM, and also a lack of readibly available knowledge in it's use. One familiar with multiple architectures stated "set the parameters, and it just worked", where as others indicated they struggled to find their way through dependencies such as the actual bootloader to use.
+* Some operators are finding some pain points in third party integrations around automation. The community likely needs to consider loosing some aspects, and some of the pain points which arose during the discussion were already known and in progress.
+* Larger operator need a means for information sharing. Not all of it is Ironic or ironic specific, a lot of it is just the complexity and intertwined dependencies which are outside of the project's ability to influence.
+<br>yy
+Hopefully, soon in the future, we will be able to have the SIG meet again, perhaps at the next [Project Teams Gathering](https://openinfra.dev/ptg/), which will be in Columbus, Ohio on October 17th-20th.
```

Comment:
it's not happening in person, so we can't have an in-person meetup :(

Reply:
Gah, yeah. :(